lib/instrumental/agent.rb

```diff
@@ -42,18 +44,28 @@ def initialize(api_key, options = {})
 
       # defaults
       # host:        collector.instrumentalapp.com
-      # port:        8000
+      # port:        8001
       # enabled:     true
       # synchronous: false
+      # secure:      true
+      # verify:      true
       @api_key         = api_key
       @host, @port     = options[:collector].to_s.split(':')
       @host          ||= 'collector.instrumentalapp.com'
-      @port            = (@port || 8000).to_i
+      desired_secure   = options[:secure].nil? ? allows_secure? : !!options[:secure]
+      if !allows_secure? && desired_secure
+        @logger.warn "Cannot connect to Instrumental via encrypted transport, SSL not available"
+        desired_secure = false
```

Comment:
I don't think I want to block secure protocol to improve our problem signaling, but I do want to say that it strikes me as particularly silly that we built a sophisticated system to tell you when important things about your software aren't working and we don't use it to notify our customers when they're having problems with our software.

So, you know, stow that away. For whenever. The Future™.